libcxx/test/std/numerics/c.math/hermite.pass.cpp

```diff
@@ -26,7 +26,12 @@
 
 #include "type_algorithms.h"
 
-inline constexpr unsigned g_max_n = 128;
+inline constexpr unsigned g_max_n =
```

Comment:
@PaulXiCao Can you review this please?

Reply:
Sure thing. Will probably not have time before the weekend.